monitoring/grafana/grafana-route.yaml

```diff
@@ -0,0 +1,15 @@
+kind: Route
+apiVersion: route.openshift.io/v1
+metadata:
+  name: grafana-ui
+spec:
+  to:
+    kind: Service
+    name: grafana
+    weight: 100
+  port:
+    targetPort: service
+  wildcardPolicy: None
+  tls:
+    termination: edge
+    insecureEdgeTerminationPolicy: Redirect
```

Comment:
Reviewers will see less red if this file has a terminal linebreak.

Reply:
Fixed!